TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/bot/BotActivity.java

```diff
@@ -174,10 +174,15 @@ protected void onStop() {
             unbindService(serviceConnection);
             boundToMovementService = false;
         }
-        presenter.stopPresenting();
         super.onStop();
     }
 
+    @Override
+    protected void onDestroy() {
```

Comment:
Yes exactly. We want it to be available as long as the activity is bound in the activity backstack. If the phone runs out of memory / the activity is killed by the user then everything is dropped, including the service.